### DISCUSSION TITLE
Use redis-stack-server as base image

### DISCUSSION THREAD

**Discussion by svabra** [OT-CONTAINER-KIT/redis-operator#776] [Polls]
We found many users wanting to use opstree's redis operator to run the RedisTimeseries, RedisJSON, RedisSearch, RedisBloom, etc.

I'd like to understand who would prefer the base image to be exchanged by the redis-stack-server image, which includes all these features - AND - that these modules could be loaded with the operator's config optionally. Also, it should allow to pass arguments to the loading of the modules. This in contrast of using the slim redis images without the additional modules. 

FYI: The license is not an obstacle. See the [RSAL](https://redis.io/docs/about/about-stack/#redis-stack-license)

**Comment by laurentpellegrino** [comment]
Any updates?
